docs/source/error_code_list2.rst

```diff
@@ -484,11 +484,12 @@ Example:
 
 .. _code-mutable-override:
 
-Check that overrides of mutable attributes are safe
----------------------------------------------------
+Check that overrides of mutable attributes are safe [mutable-override]
+----------------------------------------------------------------------
 
-This will enable the check for unsafe overrides of mutable attributes. For
-historical reasons, and because this is a relatively common pattern in Python,
+Introduced in Mypy version 1.8,
```

Comment:
I don't think we need to mention the version, unless perhaps if you want to go through and do it for all the other codes. The docs already prominently say the version at the top, and users should know that if they use an older version, they should consult that version's documentation.

Reply:
Okay sounds good! Yeah I had added it because I was initially using 1.7.1, and it wasn't there. However, I agree with your logic too, so will remove

Reply:
Pushed that change

Reply:
Thanks!